packages/hash/api/src/model/util.ts

```diff
@@ -320,6 +327,18 @@ export const generateWorkspaceEntityTypeSchema = (
     .filter(({ required }) => !!required)
     .map(({ baseUri }) => baseUri);
 
+  const links: EntityType["links"] = params.outgoingLinks.reduce(
+    (prev, { versionedUri, destinationVersionedUri }) => ({
+      ...prev,
+      [versionedUri]: { $ref: destinationVersionedUri },
+    }),
+    {},
+  );
+
+  const requiredLinks = params.outgoingLinks
+    .filter(({ required }) => !!required)
```

Comment:
Similar comment as before about convention. Would `required != null` or something not be more readable?

Reply:
I think in this situation I'd prefer not to change it. Although it would result in the correct behaviour, in my opinion it's harder to read whether `!=` or `!==` is being used, and to reason why `!=` is the correct choice of `!==` in this situation. The visual difference between `!!` and `!==` is much easier to spot on the other hand.

If you do feel strongly about it, we can change it. But I foresee this is going to be something that comes up more often, as we currently make very little use of `!=` in JS because it's so easy to confuse it with `!==`.

Reply:
Have googled. Resolving this

Reply:
(Posted that before Github showed me your comment)

I'm not sure I agree about the difference between `!!` and `!=`, I feel like `!!` is super ridiculously magical with its implicit cast to a boolean which does all of JavaScript's fuckery with falsey values. But I'll let convention decide this, hence resolving.